itests/src/test/java/fi/metatavu/edelphi/test/mock/DelfoiBulletinMocker.java

```diff
@@ -0,0 +1,66 @@
+package fi.metatavu.edelphi.test.mock;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class DelfoiBulletinMocker extends AbstractMocker {
+
+  private List<Long> bulletinIds = new ArrayList<>();
+    
+  public DelfoiBulletinMocker mock() {
+    return this;
+  }
+  
+  public DelfoiBulletinMocker cleanup() {
+    for (Long bulletinId : bulletinIds) {
+      deleteBulletin(bulletinId);
+    }
+    
+    super.cleanup();
+    
+    return this;
+  }
+
+  public Long getBulletinId(int index) {
+    Long result = bulletinIds.get(index);
+    assertNotNull(result);
+    return result;
+  }
+  
+  public long createBulletin(String title, String message, Long creatorId, Date created, Boolean important, Date importantEnds) {
+    return createBulletin(title, message, creatorId, created, creatorId, created, important, importantEnds, Boolean.FALSE);
+  }
+  
+  public long createBulletin(String title, String message, Long creatorId, Date created, Long lastModifierId, Date lastModified, Boolean important, Date importantEnds, Boolean archived) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00107)